README.md

```diff
@@ -155,13 +155,21 @@ const sortedArrayDescending = CalendarDate.sortDescending([new CalendarDate('202
 Returns the formatted string based on a provided pattern.
 
 ```typescript
-const date1 = new CalendarDate('2020-06-01');
-date1.format('dd.MM.yy');     // 01.06.20
-date1.format('d-M-yy');       // 1.6.20
-date1.format('yyyy/dd/MM');   // 2020/01/06
+const date = new CalendarDate('2020-06-01');
+date.toFormat('dd.MM.yy');     // 01.06.20
+date.toFormat('d-M-yy');       // 1.6.20
+date.toFormat('yyyy/dd/MM');   // 2020/01/06
 
 ```
 
+You can also supply a locale and options object from the Intl DateTimeFormat Api.
```

Comment:
maybe you can add a link https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat for the docs. And maybe wo should mention that the user have to take care about the browser support of the Intl DateTimeFormat Api. Or do you think it not necessary?